### ISSUE TITLE
[feature/storage]Automated post-database creation scripts

### ISSUE THREAD

**Issue by hamzabouissi** [aws/copilot-cli#3743]
Provide a lambda  backed custom-resource to run user migrations on a database
allow user to pass migrations scripts or folder to run after database get created

**Comment by huanjani** [comment]
Hello, @hamzabouissi! 
Thank you for this request. Is this something you would like to see as a pipeline action, as in https://github.com/aws/copilot-cli/issues/2992?

**Comment by hamzabouissi** [comment]
No, what I meant is, sometimes you want your database to be filled with  data before your application use it.
Those types of scripts run once after database creation, here is a blog describing the functionality:
[automate-post-database-creation-scripts-or-steps-in-an-amazon-rds-for-oracle-database](https://aws.amazon.com/blogs/database/automate-post-database-creation-scripts-or-steps-in-an-amazon-rds-for-oracle-database/)

**Comment by huanjani** [comment]
Ah. This sounds like something you might use `task run` for: https://aws.github.io/copilot-cli/docs/commands/task-run/
